torch/csrc/jit/passes/quantization.cpp

```diff
@@ -304,32 +319,52 @@ void InsertObserversImpl(
         if (!values_to_skip.count(v) && valueNeedsToBeQuantized(v)) {
           values_to_observe.push_back(v);
         }
+        if (v->node()->kind() == prim::GetAttr &&
+            v->node()->s(attr::name) == "weight") {
+          weight_values.emplace(v);
+        }
         if (v->node()->kind() == prim::CallMethod) {
           // If we find a call to a method of a child module,
           // we'll recursively insert observers for the forward function to
           // the child module.
           auto module_instance = v->node()->inputs()[0];
           auto module_method_name = v->node()->s(attr::name);
+          // TODO: looks like this block is not related to v? maybe we should
```

Comment:
The code below uses `v-node()`. I'm not sure this TODO comment is correct.

Reply:
I mean we don't need `v`, we just need `node`